astroid/protocols.py

```diff
@@ -51,6 +51,10 @@
 else:
     from typing_extensions import Literal
 
+AssignedStmtsPossibleNode = Union[
+    nodes.List, nodes.Tuple, nodes.AssignName, nodes.AssignAttr, None
+]
```

Comment:
Do we need it here or can it be imported from `astroid.nodes`?

Reply:
We're already importing `nodes` from `astroid` so I thought it would be better to use it in a similar way as we do in `pylint`, ie. `List` is `typing.List` and `nodes.List` is `astroid.nodes.List`.

Reply:
What I meant is that we define `AssignedStmtsPossibleNode` in `astroid.nodes` already. Can't we reuse it in `astroid/protocols`?

Reply:
Ah sorry! Does it make sense to creating `astroid/typing.py` as we did for `pylint`?

Reply:
I would not recommend it at this stage. Such a file only really makes sense for complex type definitions like NamedTuples or TypedDicts. This is just a simple type alias used in two files.

Reply:
See https://github.com/PyCQA/astroid/pull/1249/commits/d095f3202bac96c603460953d1824178c46188df